osu.Game/Rulesets/Edit/DistancedHitObjectComposer.cs

```diff
@@ -168,5 +181,26 @@ public ExpandingToolboxContainer()
                 FillFlow.Spacing = new Vector2(10);
             }
         }
+
+        private class DistanceSpacingToast : Toast
+        {
+            private readonly ValueChangedEvent<double> change;
+
+            public DistanceSpacingToast(LocalisableString value, ValueChangedEvent<double> change)
+                : base(getAction(change).GetLocalisableDescription(), value, string.Empty)
+            {
+                this.change = change;
+            }
+
+            [BackgroundDependencyLoader]
+            private void load(OsuConfigManager config)
+            {
+                ShortcutText.Text = config.LookupKeyBindings(getAction(change)).ToUpper();
+            }
+
+            private static GlobalAction getAction(ValueChangedEvent<double> change) => change.NewValue - change.OldValue > 0
+                ? GlobalAction.EditorIncreaseDistanceSpacing
+                : GlobalAction.EditorDecreaseDistanceSpacing;
```

Comment:
I guess this works okay. Feels... interesting that it shows the "decrease" / "increase" when adjusting using the slider, but I don't necessarily think it's a bad thing as it exposes the two shortcuts.

Reply:
UI scale slider does this, too, and so does the mania scroll speed one. So it's consistent at least.